projects/plugins/jetpack/extensions/blocks/ai-assistant/extensions/jetpack-contact-form/components/ai-assistant-popover/index.tsx

```diff
@@ -62,7 +66,24 @@ export const AiAssistantPopover = ( {
 	const { isVisible, hide, toggle, popoverProps, inputValue, setInputValue, width } =
 		useContext( AiAssistantUiContext );
 
-	const { requestSuggestion, requestingState } = useAiContext();
+	const { requestSuggestion, requestingState, eventSource } = useAiContext();
+
+	const stopSuggestion = useCallback( () => {
+		if ( eventSource ) {
+			debug( 'Stopping suggestion' );
+			eventSource?.close();
+		}
+	}, [ eventSource ] );
```

Comment:
Super minor suggestion. To be consistent with the similar code chunks (for instance, the below one):

```suggestion
	const stopSuggestion = useCallback( () => {
		if ( ! eventSource ) {
			return;
		}
		debug( 'Stopping suggestion' );
		eventSource?.close();
	}, [ eventSource ] );
```